base/src/main/java/io/spine/protobuf/MessageField.java

```diff
@@ -46,7 +46,7 @@
     private static final String GETTER_METHOD_PREFIX = "get";
 
     /**
-     * By convention underscore is used for separating words in field names
+     * By convention underscore it is used for separating words in field names
```

Comment:
This does not read to me. Under “underscore” we mean this character `_`. It may be: “By convention, the underscore character is used...”